### DISCUSSION TITLE
Can I load custom MIB files into lldpd?

### DISCUSSION THREAD

**Discussion by Ekmansoft** [lldpd/lldpd#698] [Q&A]
Can I load custom MIB files into lldpd, to get improved text decoding of neighbors etc?

It would be nice if we could also use these (custom TLVs) from the C API...

Is this possible?

**Comment by vincentbernat** [comment]
No, this is not possible (for decoding). For the C API, this is already possible: anything lldpcli can do, the C API can do.

**Reply by Ekmansoft** [reply]
Thanks Vincent.

Are you saying it is possible to parse and use MIB files already from the lldpcli?

/ Kenneth

**Reply by vincentbernat** [reply]
No, it is not. I thought you were asking if it was possible to get/set custom TLVs from the C API.

**Reply by Ekmansoft** [reply]
Ok, thanks for the clarification.
